File: src/main/java/de/floribe2000/warships_java/direct/api/stats/WeaponStatsWithShip.java
package de.floribe2000.warships_java.direct.api.stats;

/**
 * Base weapon stats container for all weapon types.
 *
 * @author SirLefti
 */
public interface WeaponStatsWithShip {

	int getFrags();
	int getMax_frags_battle();
	long getMax_frags_ship_id();
}

File: src/main/kotlin/de/floribe2000/warships_java/direct/api/stats/WeaponStatsWithShip.kt
package de.floribe2000.warships_java.direct.api.stats

/**
 * Base weapon stats container for all weapon types.
 *
 * @author SirLefti
 */
@Suppress("PropertyName", "PropertyName")
interface WeaponStatsWithShip : WeaponStats {
    val max_frags_ship_id: Long?
}